tests/test_serializer_nested.py

```diff
@@ -302,3 +305,55 @@ class Meta:
             'serializer `tests.test_serializer_nested.DottedAddressSerializer`, '
             'or set `read_only=True` on dotted-source serializer fields.'
         )
+
+
+@pytest.mark.skipif('not postgres_fields')
```

Comment:
Apparently, string conditions are [outdated](http://doc.pytest.org/en/latest/historical-notes.html#conditions-as-strings-instead-of-booleans), should maybe use:
`skipif(not postgres_fields, reason='psycopg2 is not installed')`.